examples/leaflet_local/index.html

```diff
@@ -0,0 +1,107 @@
+<html>
+<head>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" />
+    <link rel="stylesheet" href="/examples/site.css" />
+    <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"></script>
+    <script src="/dist/flatgeobuf-geojson.min.js"></script>
+    <script src="https://unpkg.com/json-formatter-js"></script>
+
+</head>
+<body>
+<style>
+</style>
+    <ul class="primary-navigation">
```

Comment:
I think it's helpful to have the examples link to each other... but for now that means copying and pasting around nav-menu snippets and slightly modifying them for each file to designate the active file.

Note these examples all read the local builds (src="/dist/flatgeobuf-geojson.min.js"), so if we duplicate these examples for a "hosted" version, we'd need to copy/paste/edit these around some more.

Alternatively, we could get rid of this duplication and do something slightly smarter by introducing some kind of static-site gen thing, but it'd mean introducing more tooling. Would you be comfortable with that?

I'd be happy either way, but I *do* think it's helpful to be able to navigate between the examples like this.

Reply:
Sure would be nice to have something similar to what openlayers have.

Reply:
> something similar to what openlayers have.

Do you mean like this page: https://openlayers.org/en/latest/examples/ ?

Reply:
Yeah.. it's quite nice. But we don't have to shoot for that level of UX as the first interation though.